library/eclipse-temurin

```diff
@@ -6,130 +6,112 @@ GitRepo: https://github.com/adoptium/containers.git
 GitFetch: refs/heads/main
 
 #------------------------------v8 images---------------------------------
-Tags: 8u382-b05-jdk-alpine, 8-jdk-alpine, 8-alpine
+Tags: 8u392-b08-jdk-alpine, 8-jdk-alpine, 8-alpine
 Architectures: amd64
-GitCommit: f308aa1a9dc0ac8775662e4c4d71840088ab076c
+GitCommit: 376294c245eba65108cee487ce209f39c15e84c8
 Directory: 8/jdk/alpine
-File: Dockerfile.releases.full
 
-Tags: 8u382-b05-jdk-focal, 8-jdk-focal, 8-focal
+Tags: 8u392-b08-jdk-focal, 8-jdk-focal, 8-focal
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: f308aa1a9dc0ac8775662e4c4d71840088ab076c
+GitCommit: 376294c245eba65108cee487ce209f39c15e84c8
 Directory: 8/jdk/ubuntu/focal
-File: Dockerfile.releases.full
 
-Tags: 8u382-b05-jdk-jammy, 8-jdk-jammy, 8-jammy
-SharedTags: 8u382-b05-jdk, 8-jdk, 8
+Tags: 8u392-b08-jdk-jammy, 8-jdk-jammy, 8-jammy
+SharedTags: 8u392-b08-jdk, 8-jdk, 8
 Architectures: amd64, arm32v7, arm64v8, ppc64le
-GitCommit: f308aa1a9dc0ac8775662e4c4d71840088ab076c
+GitCommit: 376294c245eba65108cee487ce209f39c15e84c8
 Directory: 8/jdk/ubuntu/jammy
-File: Dockerfile.releases.full
 
-Tags: 8u382-b05-jdk-centos7, 8-jdk-centos7, 8-centos7
+Tags: 8u392-b08-jdk-centos7, 8-jdk-centos7, 8-centos7
 Architectures: amd64, arm64v8, ppc64le
-GitCommit: f308aa1a9dc0ac8775662e4c4d71840088ab076c
+GitCommit: 376294c245eba65108cee487ce209f39c15e84c8
 Directory: 8/jdk/centos
-File: Dockerfile.releases.full
 
-Tags: 8u382-b05-jdk-ubi9-minimal, 8-jdk-ubi9-minimal, 8-ubi9-minimal
+Tags: 8u392-b08-jdk-ubi9-minimal, 8-jdk-ubi9-minimal, 8-ubi9-minimal
 Architectures: amd64, arm64v8, ppc64le
-GitCommit: f308aa1a9dc0ac8775662e4c4d71840088ab076c
+GitCommit: 376294c245eba65108cee487ce209f39c15e84c8
 Directory: 8/jdk/ubi/ubi9-minimal
-File: Dockerfile.releases.full
 
-Tags: 8u382-b05-jdk-windowsservercore-ltsc2022, 8-jdk-windowsservercore-ltsc2022, 8-windowsservercore-ltsc2022
-SharedTags: 8u382-b05-jdk-windowsservercore, 8-jdk-windowsservercore, 8-windowsservercore, 8u382-b05-jdk, 8-jdk, 8
+Tags: 8u392-b08-jdk-windowsservercore-ltsc2022, 8-jdk-windowsservercore-ltsc2022, 8-windowsservercore-ltsc2022
+SharedTags: 8u392-b08-jdk-windowsservercore, 8-jdk-windowsservercore, 8-windowsservercore, 8u392-b08-jdk, 8-jdk, 8
 Architectures: windows-amd64
 GitCommit: a076df4eb8dd8a60f52fbb60d9adc8e5f86c8b1a
```

Comment:
The `GitCommit` on the jre/jdk 8 windows images hasn't changed so the tools can't find the `Dockefile` in the commit (which is why the "Diff Comment" job is failing).